### ISSUE TITLE
Grid combobox example is not a good use for grid

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-practices#1281]
Related: https://github.com/w3c/aria-practices/pull/1255

The [Grid Combobox example](https://w3c.github.io/aria-practices/examples/combobox/grid-combo.html) has a dropdown with a grid where you can navigate between cells. But selecting any of the two cells for one row results in the same action: the value of the first cell is chosen. Therefore, I think the ability to navigate between cells only serves to confuse users, and the dropdown should really be a `listbox`.

However, it would still be useful to have an example that uses grid where that is more appropriate.

The first thing I thought about was a date picker. However, that is a separate widget altogether.

Another idea is a seat picker, when booking a bus ticket, train ticket, or plane ticket. It could be possible to type the seat you want (e.g. `15c`), but also a popup can show all possible seats and allow you to navigate between them and select one.

There are a few considerations with such an example:

* Maybe a seat picker shouldn't be a popup of a combobox, but its own widget?
* Seats that are unavailable should not be selectable, but should it be possible to focus them?
* Seats usually have more information than just the seat's "identifier", like which price class, if it's an emergency exit row, if it's a window seat or an aisle seat, etc.

**Comment by smhigley** [comment]
I agree that the current example isn't very illustrative of data that would benefit from being in a grid, but I think in principle it's a valid use of the pattern. Even if any cell within a row performs the same selection, there are some cases where it is more user-friendly to expose the information in the dropdown as a grid. A better example might be selecting a cloud VM image where each choice has information about size, region, OS, status, etc. It's much easier to browse and understand the choices if they are presented as grid data.

More examples are never bad though, so perhaps a train or plane seat picker might be a good second example for a grid combo?
